services/supervisord/devcontainer_test.go

```diff
@@ -40,8 +41,14 @@ import (
 
 // TODO move tests to other files and remove this one.
 func TestDevContainer(t *testing.T) {
+	if os.Getenv("DEVCONTAINER") == "" {
```

Comment:
It was removed in devcontainer PR: https://github.com/percona/pmm-managed/pull/318/files#diff-727810a1537d6d8ff464d26f15896a18L42-L45

Please check other places where the merge was incorrect.